packages/angular/ssr/public_api.ts

```diff
@@ -13,3 +13,15 @@ export {
   getOrCreateAngularAppEngine,
   destroyAngularAppEngine,
 } from './src/app-engine';
+
+export {
+  type SSGFallback,
+  type RenderMode,
+  type ServerRoute,
+  type ServerRouteAppShell,
+  type ServerRouteCSR,
+  type ServerRouteSSG,
+  type ServerRouteSSR,
+  type ServerRouteCommon,
```

Comment:
I decided to export all of these so they become part of the API goldens. Otherwise, they wouldn't be included in the golden set, which increases the risk of unintentionally introducing breaking changes.